app/controllers/articles_controller.rb

```diff
@@ -13,7 +15,8 @@ def index
   end
 
   def create
-    @article = Article.new(article_params)
+    @current_user ||= User.find(session[:user_id])
```

Comment:
if you have a `current_user` method defined in ApplicationController, you should be able to access that method here.